### PR TITLE
fixing existing mappings and adding mapping to alerts

### PR DESCRIPTION
**Pull request by aviv1ron1**

### CODE REVIEW

File: stix_shifter_modules/msatp/configuration/config.json [hunk 1]
```diff
@@ -1,43 +1,43 @@
 {
```
Comment (aviv1ron1) [inline stix_shifter_modules/msatp/configuration/config.json:1]: return to original form

File: stix_shifter_modules/msatp/stix_translation/json/from_stix_map.json [hunk 2]
```diff
@@ -53,8 +53,7 @@
         "DeviceInfo.DeviceName",
         "DeviceAlertEvents.DeviceName",
         "DeviceImageLoadEvents.DeviceName",
-        "DeviceLogonEvents.DeviceName",
```
Comment (aviv1ron1) [inline stix_shifter_modules/msatp/stix_translation/json/from_stix_map.json:56]: DeviceNetworkInfo - needs to be re-added
DeviceLogonInfo - needs to be deleted
DeviceImageLoadEvents.DeviceName - needs to be deleted

File: stix_shifter_modules/msatp/stix_translation/json/to_stix_map.json [hunk 3]
```diff
@@ -209,39 +406,115 @@
...
+        "references": [
+          "dst-ipv4-addr"
+        ],
+        "group": true
       }
     ],
-    "DeviceName": {
-      "key": "x-msatp.computer_name"
+    "OSPlatform": {
+      "key": "x-oca-asset.os_platform",
```
Comment (aviv1ron1) [inline stix_shifter_modules/msatp/stix_translation/json/to_stix_map.json:435]: change to os_name

**Comment by aviv1ron1** [comment]
This PR fixes: 

1. Existing mappings of host ip and mac data

This PR adds:
1. ms atp alerts - until now they have not been mapped. now you can receive alert data as `x-ibm-finding` objects with ttp taggings
2. add missing references to `x-oca-event` (process, url)
3. add properties to `x-oca-asset` describing the host operating system
4. add missing reference of the original event log to `x-oca-event`
5. adds links to the original report in ms atp as external refrences in `x-oca-event`

File: stix_shifter_modules/msatp/stix_translation/json_to_stix_translator.py [hunk 4]
```diff
@@ -0,0 +1,388 @@
+import re
```
Comment (delliott90) [inline stix_shifter_modules/msatp/stix_translation/json_to_stix_translator.py:1]: Why are you using your own json_to_stix_translator file? It's suppose to be generic for all JSON files. If the file `stix_shifter_utils/stix_translation/src/json_to_stix/json_to_stix_translator.py` is missing anything, perhaps your changes can be added there?

**Pull request closed without merging**